lib/segment/analytics/client.rb

```diff
@@ -20,6 +22,7 @@ def initialize attrs = {}
 
         @queue = Queue.new
         @write_key = attrs[:write_key]
+        attrs[:host] = Defaults::Request::HOST unless attrs[:host]
```

Comment:
Use the default Segment API host, unless the user provides a custom host (like we need to do to use this SDK with Polku)